src/extract.ts

```diff
@@ -133,6 +133,39 @@ export interface PytestBenchmarkJson {
     version: string;
 }
 
+type JuliaBenchmarkTrialEstimate = [
+    string,
+    {
+        params: [
+            string,
+            {
+                seconds: number;
+                samples: number;
+                evals: number;
+                overhead: number;
+                gctrial: boolean;
+                gcsample: boolean;
+                time_tolerance: number;
+                memory_tolerance: number;
+            },
+        ];
+        time: number;
+        gctime: number;
+        memory: number;
+        allocs: number;
+    },
+];
+
+type JuliaBenchmarkGroup = [
+    string,
+    {
+        data: Record<string, JuliaBenchmarkGroup | JuliaBenchmarkTrialEstimate>;
+        tags: string[];
+    },
+];
```

Comment:
Looking at the output I think the first tuple type can be 'BenchmarkGroup' explicitly and you can use the union type from above to type data

```suggestion
type JuliaBenchmarkGroup = [
    'BenchmarkGroup',
    {
        data: Record<string, JuliaBenchmark>;
        tags: string[];
    },
];
```